### ISSUE TITLE
Can't tap on tab when TalkBack is enabled

### ISSUE THREAD

**Issue by toffo10** [react-navigation/react-navigation#12446]
### Current behavior

When you enable the TalkBack feature in Android, you can't click on the tabs in the TabView anymore.
 
https://github.com/user-attachments/assets/362eec35-8e69-4e3a-92d8-5f6a6d847d88

Upon further inspection of the library, seems that removing pointerEvents="none" in TabBarItem.tsx resolves the issue:
https://github.com/react-navigation/react-navigation/blob/main/packages/react-native-tab-view/src/TabBarItem.tsx#L231

### Expected behavior

The tabs should be clickable when TalkBack is enabled.

(Video with the removal of pointerEvents="none")
https://github.com/user-attachments/assets/fb1520a6-5dfd-4adb-b42b-b1ab11681f50

### Reproduction

https://github.com/toffo10/react-native-tab-view-talkback

### Platform

- [x] Android
- [ ] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Packages

- [ ] @react-navigation/bottom-tabs
- [ ] @react-navigation/drawer
- [ ] @react-navigation/material-top-tabs
- [ ] @react-navigation/stack
- [ ] @react-navigation/native-stack
- [ ] react-native-drawer-layout
- [x] react-native-tab-view

### Environment

- [] I've removed the packages that I don't use

| package                                | version |
| -------------------------------------- | ------- |
| react-native-tab-view                  | 4.0.5
| react-native-pager-view                | 6.7.0
| react-native                           | 0.77.1 
| node                                   | 18.19.0 |

**Comment by GodlXlFT** [comment]
any update on this issue??
